third-party/realdds/readme.md

```diff
@@ -61,6 +61,8 @@ Alternatively, one can use RealDDS to bypass librealsense. This will avoid some
 
 Or you can directly interact with the DDS subsystem, using any implementation. This bypasses librealsense and RealDDS completely. While the protocol used by RealDDS (the topic structure, message formats, etc.) will likely not change much, it is not guaranteed to stay static in the face of new features or bugs.
 
+**Note**: In some systems where firewall is enabled, you may see the message "Do you want to allow networks to access this app", make sure to allow this in your network to ensure proper communication with the DDS device. This message shows up since we open and bind a socket when using DDS, and we need to allow it through the network security settings (a rule to allow our app is added to the firewall rules).
```

Comment:
I would remove the parenthesized sentence `(a rule to allow our app is added to the firewall rules)`. It might be understood that we are adding the rule, which is not the case. Lets just mention that it should be allowed in the network security settings, let the user handle how according to his specific system.

Reply:
done